scripts/publish_video/publish_dapps.js

```diff
@@ -0,0 +1,84 @@
+require('../../page_conf')
+const { mongo } = require('../../helper');
+const { getOperations, sleep, steemPostExist, tryPublish, shouldSkip } = require('./helper');
+
+(async() => {
+
+  console.log('===============================')
+
+  const videos = await mongo.Video.find({
+    status: { $in: ['publish_manual'] },
+    title: { $ne: null }
+  }).sort('-created');
+  console.log('## Videos to publish:', videos.length)
+
+  if (videos.length === 0) {
+    await sleep(5000)
+  }
+
+  for (const video of videos) {
+    const shouldWeSkip = await shouldSkip(video);
+    if (shouldWeSkip) {
+      continue;
+    }
+    console.log('===============================')
+    console.log('## Publishing Video to HIVE:', video.owner, video.permlink, ' -- ', video.title)
+    const oldDescription = video.description;
+
+    try {
+      if (!(await steemPostExist(video.owner, video.permlink))) {
+        video.description = `${video.description}\n\n[![appStore](https://i.imgur.com/enwTLng.png)](https://apps.apple.com/us/app/3speak/id1614771373) | [![GooglePlayStore](https://i.imgur.com/6K5fgGX.png)](https://play.google.com/store/apps/details?id=tv.threespeak.app) | [![Support @threespeak](https://i.imgur.com/2cEH8bp.png)](https://hivesigner.com/sign/account-witness-vote?witness=threespeak&approve=1) | [![Support @sagarkothari88](https://i.imgur.com/bTdSCuq.png)](https://hivesigner.com/sign/account-witness-vote?witness=sagarkothari88&approve=1)`;
```

Comment:
I thought we agreed to standardize the template? Also doing what you're doing here would cause the video description on 3Speak to include all this extra stuff. 

Additionally, this should only show if it's from the mobile app